### PR TITLE
Tcio 99/hero

### PR DESCRIPTION
**Pull request by julika-gulchitai**

### CODE REVIEW

File: src/widgets/HomePageWidgets/Hero/Hero.tsx [hunk 1]
```diff
@@ -27,20 +36,25 @@ export const Hero = () => {
...
+              clickable: true,
+              bulletClass: 'neutral-50',
+              bulletActiveClass: 'primary-50',
             }}
             navigation={{
               nextEl: '.button-next-slide',
               prevEl: '.button-prev-slide',
             }}
-            className="w-full h-full"
+            // className="w-full h-full"
```
Comment (BohdanTaran) [inline src/widgets/HomePageWidgets/Hero/Hero.tsx]: You should remove useless comments

File: src/widgets/HomePageWidgets/Hero/Hero.tsx [hunk 2]
```diff
@@ -87,9 +90,9 @@ export const Hero = () => {
               </button>
             )}
           </Swiper>
-          <div className="absolute z-20 bottom-[27px] left-0 w-full w-[23.5px] h-1">
-            <div className="swiper-pagination"></div>
-          </div>
+          {/* <div className="bottom-0 left-0 w-[23.5px] h-1"> */}
```
Comment (BohdanTaran) [inline src/widgets/HomePageWidgets/Hero/Hero.tsx:93]: same

**Review by BohdanTaran — APPROVED** [review]
Before merge check comments in the code. Great job!

File: src/pages/HomePage/HomePage.tsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import Carousel from '../../widgets/Carousel/Carousel';
+import { Hero } from '../../widgets/HomePageWidgets/Hero/Hero';
```
Comment (yatskivkath) [inline src/pages/HomePage/HomePage.tsx:2]: add index.ts file in Hero folder

File: src/pages/HomePage/HomePage.tsx [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import Carousel from '../../widgets/Carousel/Carousel';
```
Comment (yatskivkath) [inline src/pages/HomePage/HomePage.tsx:1]: add index.ts to Carousel folder

File: src/widgets/HomePageWidgets/Hero/components/HeroSlider.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+          className={`font-roboto font-medium text-[${mainColor}] text-xl laptop:text-2xl mb-4 laptop:mb-10 w-[335px] laptop:w-[526px]`}
+        >
+          {hero.description}
+        </div>
+
+        <button
+          type="button"
+          className={`bg-[#D5B5FF] text-[${buttonColor}] w-[130px] h-[40px] rounded-md text-sm laptop:w-[240px] laptop:h-[45px] laptop:font-medium laptop:text-xl`}
+        >
+          Shop Collection
```
Comment (yatskivkath) [inline src/widgets/HomePageWidgets/Hero/components/HeroSlider.tsx:29]: I think this button text should also be retrieved from BE (or from mocked props in your case). Potentially for different heroes, there might be different text. 

You can set 'Shop Collection' as default value e.g. `{hero.call_to_action ?? 'Shop Collection'}`

**Pull request merged**